bee-network/bee-autopeering/examples/node.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(warnings)]
+
+use bee_autopeering::{
+    config::AutopeeringConfigJsonBuilder,
+    init,
+    peerstore::{InMemoryPeerStore, SledPeerStore, SledPeerStoreConfig},
+    AutopeeringConfig, Event, Local, NeighborValidator, Peer, ServiceProtocol, AUTOPEERING_SERVICE_NAME,
+};
+
+use libp2p_core::identity::ed25519::Keypair;
+use log::LevelFilter;
+use serde_json::Value;
+use tokio::signal::ctrl_c;
+use tokio_stream::StreamExt;
+
+use std::{future::Future, io, net::SocketAddr, pin::Pin};
+
+const AUTOPEERING_VERSION: u32 = 1;
+const NETWORK_SERVICE_NAME: &str = "chrysalis-mainnet";
+const BS16_ED25519_PRIVATE_KEY: &str = "1858c941a1c4454f4df77be93481b66e5f4dcff885f30c6a6a4cb214d2fea21072e7dce6a4d4d7b89460e84bb0e3b4475b528524e7ceb741f7646400ef9f2c7b";
+
+fn setup_logger(level: LevelFilter) {
+    fern::Dispatch::new()
+        .level(level)
+        .chain(io::stdout())
+        .apply()
+        .expect("fern");
+}
+
+fn read_config() -> AutopeeringConfigJsonBuilder {
+    let config_json = r#"
+    {
+        "enabled": true,
+        "bindAddress": "0.0.0.0:14626",
+        "entryNodes": [
+            "/dns/lucamoser.ch/udp/14826/autopeering/4H6WV54tB29u8xCcEaMGQMn37LFvM1ynNpp27TTXaqNM",
```

Comment:
These entry nodes are used for IF deployments: 
https://github.com/iotaledger/hornet/blob/if-deployment/config_devnet.json#L204-L205

Maybe we can remove Luca's node.

Reply:
I am not entirely sure what's the policy here, but the nodes are supposed to be primarily by the users. So I guess we could also have a separate branch called `if-deployment` where all community entry nodes are removed. What's your take on this @thibault-martinez ?

Reply:
Let's just use the same hornet has on the dev branch https://github.com/gohornet/hornet/blob/develop/config.json#L187

Reply:
Ah, I didn't click it was an example, maybe remove Luca yeah.

Reply:
Removed from the example.